### PR TITLE
Prevent unapproved assets from viewing

### PR DESCRIPTION
**Pull request by ShikhaMishra9**
[AB#81114](https://dev.azure.com/EquinorASA/d02c350d-092d-41bf-a812-f05d5d8ad1b0/_workitems/edit/81114)

### CODE REVIEW

File: pages/api/popular.ts [hunk 1]
```diff
@@ -77,6 +77,9 @@ const PopularAssetsHandler: NextApiHandler = async (req, res) => {
       query: { name: "data product" },
     })
 
+    if (dataProductTypeIdRes.body.results[0]?.id === "undefined") {
```
Comment (wenche) [inline pages/api/popular.ts:80]: This is the one Petter suggested in the other PR?

**Review by wenche — APPROVED** [review]
LGTM 👍 

We should align with the case of no asset as well - we would probably like to return 404 if that's the case

File: pages/api/popular.ts (hunk 1 above)
Comment (ShikhaMishra9) [reply pages/api/popular.ts:80]: yes...should i put it in another PR

Comment (wenche) [reply pages/api/popular.ts:80]: It's fine by me, just curious :)

**Pull request merged**